src/main/java/net/imagej/omero/SSLUtils.java

```diff
@@ -0,0 +1,74 @@
+/*-
+ * #%L
+ * ImageJ software for multidimensional image processing and analysis.
+ * %%
+ * Copyright (C) 2013 - 2019 Open Microscopy Environment:
+ * 	- Board of Regents of the University of Wisconsin-Madison
+ * 	- Glencoe Software, Inc.
+ * 	- University of Dundee
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package net.imagej.omero;
+
+import java.security.Security;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ */
+public class SSLUtils {
+
+	/**
+	 * Ensure that anonymous cipher suites are enabled in the JRE.
+	 * OMERO 5.5 will no longer require this. See:
+	 * https://github.com/openmicroscopy/openmicroscopy/pull/5949/
```

Comment:
May can add a card to note that we should revert this in due course. (Might even want to `@Deprecate` now.)

Reply:
Added to test.sh as a pointer to this PR.